src/repository.rs

```diff
@@ -98,8 +99,7 @@ impl Repository {
         fs_ioc_enable_verity::<&OwnedFd, Sha256HashValue>(&ro_fd)?;
 
         // double-check
-        let measured_digest: Sha256HashValue = fs_ioc_measure_verity(&ro_fd)?;
-        assert!(measured_digest == digest);
+        fsverity::compare_verity(&ro_fd, &digest)?;
```

Comment:
Thanks for de-panicking this.  Could maybe use some `.context()` though (although that's also kinda pointless if we're moving away from `anyhow`...).